sssom/cliques.py

```diff
@@ -59,7 +59,17 @@ def to_networkx(msdf: MappingSetDataFrame) -> nx.DiGraph:
     return g
 
 
-def split_into_cliques(msdf: MappingSetDataFrame):
+def split_into_cliques(msdf: MappingSetDataFrame) -> List[MappingSetDocument]:
+    """Split MappingSetDataFrames into cliques.
+
+    :param msdf: MappingSetDataFrame object
+    :type msdf: MappingSetDataFrame
```

Comment:
don't add type annotations in the docs. They're already taken care of by type annotations

Reply:
Is this commonly accepted practice? My assumption was that in order for documentation systems like sphinx to generate anything meaningful it would need the type hints in the docstring.

Reply:
The [`sphinx-autodoc-typehints`](https://github.com/agronholm/sphinx-autodoc-typehints) plugin to Sphinx takes care of this (if that's not already installed on this project then I will add it now). I think it's also obvious that having the type annotations be part of the syntax gives the benefit of it being typo checked

Reply:
See #159 for the pull request where I add sphinx-autodoc-typehints, as well as a check to CI that the docs can even be built (currently there is an issue)

Reply:
Those are autogenerated by a VSCode plugin ([Python doctoring generator](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring)) that I use for every project. I can take those off if they seem redundant.

Reply:
Not only are they a redundancy, but they're also a liability, since they could get out of sync with the actual type annotations. I'd definitely suggest turning off that part of the docstring generator.